app/settings.py

```diff
@@ -99,3 +99,5 @@ def get_key(_key_name):
 
 # JWT configurations
 EQ_JWT_LEEWAY_IN_SECONDS = 120
+
+EQ_ROLE_PERMISSIONS = {'admin': ['flush']}
```

Comment:
Isn't admin a bit of a generic term?
should we name the role more appropriately to the permissions enclosed in it?